docs/devGuide/contributing.md

```diff
@@ -0,0 +1,72 @@
+
+## Things to do before getting started
+
+Make sure you have the project set up and ready. We have guides detailing what we [expect our contributors to know]({{baseurl}}/devguide/devguide#requirement), our required [environment]({{baseurl}}/devGuide.html#environment) and our [development process]({{baseurl}}/devGuide.html#development-process).
+
+## How to contribute
+
+All contributions are welcome. You can contribute to MarkBind by:
+- Reporting bugs
+- Requesting features
+- Making changes (with a corresponding Pull Request), by:
+   - Updating documentation
+   - Fixing bugs
+   - Implementing features 
+
+### How to report a bug
+
+Report bugs in our [issue tracker](https://github.com/MarkBind/markbind/issues), following the guideslines below. Following the guidelines helps us in undestanding your report better and reproduce the behavior more easily.
+
+- Has your bug been raised up before? Before reporting a bug, please search if it has been already reported in the existing issues.
+- Is your bug from the current version or an older version of MarkBind? What OS are you on? Tell us about your environment, for example, the MarkBind version you are on.
+- What did you do? What did you expect to happen? What actually happened? 
+  - Include the actual MarkBind code causing the issue, the steps for reproduction, the actual, raw output, and what you expect instead. This helps us to reproduce the bug to understand it better. 
+  - You are also encouraged to submit a PR that reproduces this in `test\functional\test_site\bugs\`.
+
+Please do not submit personal support requests as bugs, [submit a blank issue](https://github.com/MarkBind/markbind/issues/new) to our issue tracker and label it with <a href="https://github.com/MarkBind/markbind/issues?q=is%3Aopen+is%3Aissue+label%3Aquestion" class="badge" style="color:white; background-color: #A41BD6;">question</a>.
+
+### How to request a feature
+
+Request for new features or enhancements to existing features in our [issue tracker](https://github.com/MarkBind/markbind/issues), following the guideslines below. Following the guidelines helps us in undestanding what your problem or use case is, so that contributors may know exactly what feature to implement.
+
+- Has someone already requested your feature before? Before requesting a new feature, please search if it has been already requested in the existing issues.
+- Is your request related to a problem? Provide a clear and concise description of what the problem is. Eg: I have an issue when [...]
+- Describe the solution you'd like. Provide a clear and concise description of what you want to happen.
+- Describe alternatives you've considered. Let us know about other solutions you've tried or researched.
+- If any, share some additional context. Is there anything else you can add about the proposal? You might want to link to related issues here if you haven't already.
```

Comment:
`You can also share some additional context ... e.g. ...`
`You might want to link to related issues here if you haven't already.` is for when you're writing the request template, so let's ~remove~ it here